lib/pbench/test/unit/agent/task/test_copy_result_tb.py

```diff
@@ -67,17 +60,13 @@ def test_missing_md5(self, caplog, valid_config):
             expected_error_message = (
                 f"tarball's .md5 does not exist, '{missing_md5_tar.name}.md5'"
             )
-            try:
+            with pytest.raises(FileNotFoundError) as e:
                 crt = CopyResultTb(
                     "controller", missing_md5_tar.name, self.config, self.logger
                 )
-                crt.copy_result_tb()
-            except SystemExit:
-                assert caplog.records
-                assert len(caplog.records) == 1
-                assert caplog.records[0].message == expected_error_message
-            else:
-                assert False
+                crt.copy_result_tb("token")
+            assert str(e).endswith(expected_error_message)
```

Comment:
Again, didn't you capitalize the message? Is this actually doing a caseblind match, or is it failing?

Reply:
Sorry, I Forgot